Content.Shared/Humanoid/Markings/MarkingColoring.cs

```diff
@@ -26,12 +26,12 @@ public static class MarkingColoring
     /// <summary>
     ///     Returns list of colors for marking layers
     /// </summary>
-    public static List<Color> GetMarkingLayerColors
-    (
+    public static List<Color> GetMarkingLayerColors(
         MarkingPrototype prototype,
         Color? skinColor,
         Color? eyeColor,
-        MarkingSet markingSet
+        MarkingSet markingSet,
+        List<string>? ignores = null
```

Comment:
Floof comments + probably revert the parenthesis movement since it's conflict bait.